src/routes/safe/container/selector.ts

```diff
@@ -1,19 +1,22 @@
 import { List, Map } from 'immutable'
 import { createSelector } from 'reselect'
 
+import { Token } from 'src/logic/tokens/store/model/token'
 import { tokensSelector } from 'src/logic/tokens/store/selectors'
 import { getEthAsToken } from 'src/logic/tokens/utils/tokenHelpers'
-import { isUserOwner } from 'src/logic/wallets/ethAddresses'
+import { isUserAnOwner } from 'src/logic/wallets/ethAddresses'
 import { userAccountSelector } from 'src/logic/wallets/store/selectors'
 
 import { safeActiveTokensSelector, safeBalancesSelector, safeSelector } from 'src/routes/safe/store/selectors'
-import { Token } from 'src/logic/tokens/store/model/token'
+import { SafeRecord } from '../store/models/safe'
```

Comment:
absolute path?

Reply:
this one is shorter :)

Reply:
neh, I don't think that is a valid criterion. In my case, I'm trying to only use relative paths for those imports that are in the same directory, the rest should be absolute.

We should standardize this somehow.